contextualized/dags/torch_notmad/torch_notmad.py

```diff
@@ -0,0 +1,286 @@
+#define Encoder
+import torch
+from torch import nn
+import pytorch_lightning as pl
+import numpy as np
+from contextualized.functions import identity_link
+torch.set_default_tensor_type(torch.FloatTensor)
+
+#local imports
+from graph_utils import project_to_dag_torch
+from torch_utils import DAG_loss, NOTEARS_loss
+
+class NGAM(nn.Module):
```

Comment:
Remove and use the modules.py NGAM or MLP as the default encoder.

Reply:
Would it be good to keep this as is until we update the explainer and ngam of modules.py?